### PR TITLE
Update BCD info, part 14

### PR DESCRIPTION
**Pull request by OnkarRuikar**
The PR updates BCD info in WebAPI docs.
It focuses on files that have only left behind 'experimental' tags and headers.

### CODE REVIEW

File: files/en-us/web/api/deprecationreportbody/anticipatedremoval/index.md [hunk 1]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Property
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/deprecationreportbody/anticipatedremoval/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/deprecationreportbody/columnnumber/index.md [hunk 2]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Property
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/deprecationreportbody/columnnumber/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/deprecationreportbody/id/index.md [hunk 3]
```diff
@@ -7,11 +7,10 @@ tags:
   - Property
   - Reference
   - id
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/deprecationreportbody/id/index.md:10]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/deprecationreportbody/index.md [hunk 4]
```diff
@@ -5,13 +5,12 @@ page-type: web-api-interface
 tags:
   - API
   - DeprecationReportBody
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/deprecationreportbody/index.md:8]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/deprecationreportbody/linenumber/index.md [hunk 5]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Property
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/deprecationreportbody/linenumber/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/deprecationreportbody/message/index.md [hunk 6]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Property
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/deprecationreportbody/message/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/deprecationreportbody/sourcefile/index.md [hunk 7]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Property
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/deprecationreportbody/sourcefile/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/deprecationreportbody/tojson/index.md [hunk 8]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Method
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/deprecationreportbody/tojson/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/interventionreportbody/columnnumber/index.md [hunk 9]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Property
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/interventionreportbody/columnnumber/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/interventionreportbody/id/index.md [hunk 10]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Property
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/interventionreportbody/id/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/interventionreportbody/index.md [hunk 11]
```diff
@@ -4,14 +4,13 @@ slug: Web/API/InterventionReportBody
 page-type: web-api-interface
 tags:
   - API
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/interventionreportbody/index.md:7]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/interventionreportbody/linenumber/index.md [hunk 12]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Property
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/interventionreportbody/linenumber/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/interventionreportbody/message/index.md [hunk 13]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Property
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/interventionreportbody/message/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/interventionreportbody/sourcefile/index.md [hunk 14]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Property
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/interventionreportbody/sourcefile/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

File: files/en-us/web/api/interventionreportbody/tojson/index.md [hunk 15]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Method
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/interventionreportbody/tojson/index.md:9]: This was correct. BCD is incorrect `DeprecationReportBody.*` are only supported by one engine, Chromium, and should be flagged as _experimental_ there.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Some errors in bcd

**Comment by OnkarRuikar** [comment]
Empty commit got pushed continuing in #19577.
Remaining files are the same so no need to check again.

**Pull request closed without merging**